Bottom: 817d1e90c9b9233b5955ed861405baf78b7ed41f
Top:    0be364b3b12708c3744e5fc6d9474da399d6c326
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-21 22:15:35 +0000

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index fdaeafb..5db9b1d 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1646,7 +1646,7 @@ fill_board_outline (hidGC gc, const BoxType *drawn_area)
   polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
 //  hid_draw_fill_pcb_polygon (gc, &polygon, drawn_area);
-  hid_draw_thindraw_pcb_polygon (gc, &polygon, drawn_area);
+  hid_draw_thin_pcb_polygon (gc, &polygon, drawn_area);
   poly_FreeContours (&polygon.NoHoles);
 }
